src/tiledb/cloud/geospatial/ingestion.py

```diff
@@ -0,0 +1,1182 @@
+import math
+import os
+from dataclasses import dataclass
+from typing import Any, Iterable, List, Mapping, Optional, Sequence, Tuple, Union
+
+import fiona
+import laspy
+import rasterio
+import shapely
+from rtree import index
+
+import tiledb
+from tiledb.cloud.utilities import Profiler
```

Comment:
The `Profiler` placeholders look good. When you're ready to use the profiler, `tiledb.cloud.utilities.create_log_array` will create the log array, which is needed by the functions being profiled.

Reply:
👍